### PR TITLE
Add parameters and last fixes before launch

### LINKED ISSUE #372: US-specific social media tags on policyengine.org/us
Currently previews as UK

https://socialsharepreview.com/?url=https://policyengine.org/us

### LINKED ISSUE #597: Remove snapshot from PolicyEngine US
Until we verify and test historical and future parameters.

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Fixes #597 
Fixes #372

### CODE REVIEW

File: policyengine-client/src/countries/us/us.jsx [hunk 1]
```diff
@@ -82,6 +102,32 @@ export class US extends Country {
...
+            },
+            "Payroll taxes": {
+                "Medicare": [
+                    "medicare_rate",
+                    "additional_medicare_rate",
+                ],
+                "Social Security": [
+                    "social_security_tax_threshold",
+                    "social_security_tax_rate",
+                ],
```
Comment (MaxGhenis) [inline policyengine-client/src/countries/us/us.jsx:130]: Could you organize benefits by agency while in here? https://github.com/PolicyEngine/policyengine/issues/598

File: policyengine-client/src/countries/us/us.jsx [hunk 2]
```diff
@@ -82,6 +102,32 @@ export class US extends Country {
...
+                    "American Opportunity Credit": [
+                        "abolish_aoc",
+                        "aoc_refundable_percentage",
+                    ],
+                },
+            },
+            "Payroll taxes": {
+                "Medicare": [
+                    "medicare_rate",
+                    "additional_medicare_rate",
```
Comment (MaxGhenis) [inline policyengine-client/src/countries/us/us.jsx:125]: Remove rates or first make them employee-only in openfisca-us https://github.com/PolicyEngine/openfisca-us/issues/643

File: policyengine-client/src/countries/us/us.jsx [hunk 3]
```diff
@@ -259,6 +313,9 @@ export class US extends Country {
             ],
             "Demographics": [
                 "age",
+                "is_in_k12_school",
+                "is_full_time_student",
```
Comment (MaxGhenis) [inline policyengine-client/src/countries/us/us.jsx:317]: https://github.com/PolicyEngine/openfisca-us/issues/644 first

File: policyengine-client/src/countries/us/us.jsx [hunk 4]
```diff
@@ -64,6 +64,26 @@ export class US extends Country {
...
             ],
             "Credits": {
+                "Child tax credit": {
+                    "Eligibility": [
+                        "ctc_child_age",
+                        "ctc_child_young_age",
+                    ],
+                    "Amount": [
+                        "ctc_child",
+                        "ctc_child_young_bonus",
```
[the remark this answers is not in the document]
Comment (nikhilwoodruff) [reply policyengine-client/src/countries/us/us.jsx]: Yep, removed

**Pull request merged**